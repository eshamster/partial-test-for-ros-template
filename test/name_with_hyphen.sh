#!/bin/bash

set -eu

test_name=test-name-with-hyphen

function ros_t()
{
    ros template $@
}

ros_t deinit ${test_name} > /dev/null 2>&1 && :
ros_t init ${test_name}
ros_t checkout ${test_name}
ros_t checkout 2>&1 | grep -E "(${test_name}|default)" && :
ros_t deinit ${test_name}
