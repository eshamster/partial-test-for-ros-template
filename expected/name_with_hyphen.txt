current default is "test-name-with-hyphen"
default
test-name-with-hyphen
